tests/installation/change_desktop.pm

```diff
@@ -31,13 +31,6 @@ sub change_desktop {
         wait_screen_change { send_key 'ret'; };
     }
 
-    if (check_screen('dependency-issue', 5) && get_var("WORKAROUND_DEPS")) {
```

Comment:
We cannot just remove this code =(
Needle is used here: https://openqa.suse.de/tests/2777407#step/installation_overview_before/9

Reply:
installation_overview_before has been removed. according to the changes any dependency will be solved before this point as the resolve_dependency_issues will run before the change_desktop. WORKAROUND_DEPS and BREAK_DEPS are handling inside the resolve_dependency_issues.  is it possible to dependency-issue in this point again?

Reply:
That's not what I'm saying. Removing this code from this module is very appreciated, but either needles have to be adjusted as later on it's just `manual-intervention`. Or to split the work we can move this part of code to that module to play it safe. But just removing might break things, as I'm not sure that `manual-intervention` needle will match `dependency-issue` cases...

Reply:
Actually this was my first approach. i have edited and i added the deleted block in the resolve_dependency_issues.pm as it is the most easy way to solve this.